### DISCUSSION TITLE
Screen share is tiny

### DISCUSSION THREAD

**Discussion by abedi-h** [mynaparrot/plugNmeet-server#224] [General]
Hi

I just made a test from your Demo. everything seems perfect except the screen sharing. it shows to me very tiny and contained. is there a way to make it bigger?

I attached the picture
![brave_MS4ooF6MKq](https://user-images.githubusercontent.com/66095125/224343073-2ff37a3b-83f8-4f91-a406-3f41f45f2c58.png)

**Comment by jibon57** [comment]
That's from presenter end only. From other side user will see full screen. We are doing this to reduce mirror effect from presenter side.

**Reply by abedi-h** [reply]
OK
Thanks for your blazing fast response 👍 
Really appreciate your great work

**Reply by german2209** [reply]
Hi, 
I'm testing PlugNMeet, and found that if I perform a local recording, and I share my 2nd. monitor full screen, I can't get a nice recording of the meeting, because if I record the 2nd monitor the presenter video will be missing, and if I record the main screen, the shared screen is recorded in the tiny sqare.
How can I get the Presenter and the shared screen together in the same recording?

**Reply by jibon57** [reply]
Better to use plugNmeet recorder. Local recording doesn't work perfectly.
